src/main/java/com/spotinst/sdkjava/model/SpotOceanEcsClusterClient.java

```diff
@@ -0,0 +1,141 @@
+package com.spotinst.sdkjava.model;
```

Comment:
Create a 'client' folder under model and move this file under it in correct hierarchy.